lib/grover/js/processor.js

```diff
@@ -124,6 +124,18 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       page.setGeolocation(geolocation);
     }
 
+    const failOnRequest = options.requestFailure; delete options.requestFailure;
+    if (failOnRequest !== undefined) {
```

Comment:
this would still enable the error reporting if `requestFailure: ''` or `requestFailure: false` which would not appear to be the intent. Suggest:
```suggestion
    if (failOnRequest) {
```